### ISSUE TITLE
Dropdown Input: Minimal

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#696]
**Issue:**
1. No focus control indicator.

**Solution:**
1. Add focus control indicator.

**Comment by TheMooseman** [comment]
I think this is already working, tabbing through the minimal storybook shows focus indicators on the outside of the dropdowns
